hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationManager.java

```diff
@@ -493,6 +498,16 @@ public boolean isContainerReplicatingOrDeleting(ContainerID containerID) {
         .isContainerReplicatingOrDeleting(containerID);
   }
 
+  private ECContainerReplicaCount getECContainerReplicaCount(
+      ContainerInfo containerInfo) throws ContainerNotFoundException {
+    Set<ContainerReplica> replicas = containerManager.getContainerReplicas(
+        containerInfo.containerID());
+    List<ContainerReplicaOp> pendingOps =
+        containerReplicaPendingOps.getPendingOps(containerInfo.containerID());
+    // TODO: define maintenance redundancy for EC
```

Comment:
Is it worth adding the Jira number here too - ([HDDS-6975](https://issues.apache.org/jira/browse/HDDS-6975) ?